backend/src/geo_helpers.rs

```diff
@@ -214,6 +214,20 @@ pub fn make_arrow(line: Line, thickness: f64, double_ended: bool) -> Option<Poly
     Some(Polygon::new(LineString::new(pts), Vec::new()))
 }
 
+pub fn thicken_line(line: Line, thickness: f64) -> Polygon {
+    let angle = angle_of_line(line);
+    Polygon::new(
+        LineString::new(vec![
+            euclidean_destination_coord(line.start, angle - 90.0, thickness * 0.5),
```

Comment:
I should put this in geo... unfortunately I have a big open [LineMeasure change in PR limbo](https://github.com/georust/geo/pull/1298) that I'd like to resolve first... I'll go kick that to see if it's still alive. 🤔